components/apps_form/apps_form_field/apps_form_select_field.tsx

```diff
@@ -16,6 +25,11 @@ export type Props = {
     value: AppSelectOption | null;
     onChange: (value: AppSelectOption) => void;
     performLookup: (name: string, userInput: string) => Promise<AppSelectOption[]>;
+    teammateNameDisplay?: string;
+    actions: {
+        autocompleteChannels: (term: string, success: (channels: Channel[]) => void, error: () => void) => (dispatch: any, getState: any) => Promise<void>;
+        autocompleteUsers: (search: string) => Promise<UserAutocomplete>;
```

Comment:
Can we change this to `autocompleteUsersInTeam` instead? We've received requests to limit the search to users in the current team.

Reply:
I'm fine with deferring this to a different PR, though this is a quick fix since we're already touching this code

Reply:
Actually, this definitely makes sense to be in another PR :+1:

Also, I was thinking it would be useful if eventually the channel selectors supported options to search in certain contexts:
- current team
- all teams
- specific team

Along with supporting a team selector, we would be able to support scoped channel searching in a form. The App developer can use the `select field refresh` described here https://developers.mattermost.com/integrate/apps/structure/forms/#select-fields